examples/tensorflow/segmentation/README.md

```diff
@@ -143,9 +143,4 @@ To export a model to the OpenVINO IR and run it using the Intel® Deep Learning
 
 ## Results
 <a name="results"></a>
-
```

Comment:
Please adjust the link in the "Run Instance Segmentation Sample" paragraph. I would recommend adding direct links to pre-trained weights for Mask-R-CNN in this README.
```
- Download the pre-trained weights in checkpoint format and provide the path to them using `--weights` flag. The link to the
archive with pre-trained weights can be found in the `TensorFlow checkpoint` column of the [results](#results) table.
```

Reply:
Done